src/pinghelper.cpp

```diff
@@ -50,17 +51,25 @@ void PingHelper::start(const QString& serverIpv4Gateway,
 
   m_gateway = serverIpv4Gateway;
   m_source = deviceIpv4Address.section('/', 0, 0);
-  m_pingSender =
+
+  if (s_has_critical_ping_error) {
+    logger.info() << "Encountered Unrecoverable ping error, using DUMMY Ping";
+    m_pingSender = new DummyPingSender(m_source, this);
```

Comment:
I like the solution, but we should go back to the normal ping sender after a while. Maybe after a few minutes?

Reply:
Good idea! Done :)